Mathlib/Topology/Algebra/InfiniteSum/Ring.lean

```diff
@@ -107,6 +107,28 @@ theorem tsum_mul_right [T2Space α] : ∑' x, f x * a = (∑' x, f x) * a := by
 theorem tsum_div_const [T2Space α] : ∑' x, f x / a = (∑' x, f x) / a := by
   simpa only [div_eq_mul_inv] using tsum_mul_right
 
+theorem HasSum.const_div (h : HasSum (1 / f) a) (b : α) :
+    HasSum (fun i ↦ b / f i) (b * a) := by
+  have := h.mul_left b
+  simp only [div_eq_mul_inv, Pi.mul_apply, Pi.one_apply, Pi.inv_apply, one_mul] at *
+  exact this
```

Comment:
Does `h.mul_left b` work here? Certainly `simpa only [mul_one_div] using h.mul_left b` should work.

Reply:
Hang on, I see the issue, you need the Pi.inv_apply etc to simplify the hypothesis, not the goal. I'd be inclined to write the hypothesis as `HasSum (fun x \mapsto 1 / f x) a` which will shorten the proof, and also will probably make it easier to verify the hypotheses when this lemma is used. (Ditto for the other lemmas here.)